python/build_definitions/bid.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) YugabyteDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations
+# under the License.
+#
+
+import os
+import subprocess
+from yugabyte_db_thirdparty.build_definition_helpers import *  # noqa
+
+
+class BidDependency(Dependency):
+    def __init__(self) -> None:
+        super(BidDependency, self).__init__(
+            name='bid',
+            version='20U2',
+            url_pattern='https://netlib.org/misc/intel/IntelRDFPMathLib{0}.tar.gz',
+            build_group=BuildGroup.CXX_UNINSTRUMENTED)
+        self.copy_sources = True
+
+        self.patches = ['bid.patch', 'bid-crlf.patch']
+
+    def get_additional_compiler_flags(self, builder: BuilderInterface) -> List[str]:
+        flags = ['-Wno-error=missing-braces']
+        return flags
+
+    def build(self, builder: BuilderInterface) -> None:
+        with PushDir("LIBRARY"):
+            builder.build_with_make(
+                dep=self,
+                extra_make_args=['_CFLAGS_OPT=-fPIC',
```

Comment:
Does just -fPIC not work?

Reply:
This is how DocumentDB built it. Do you have a strong preference on this?

Reply:
Just `-fPIC` does not work

`make: PIC: No such file or directory`